lib/responseDownloadFormats/html-aggregated/components/AggregatedTable.tsx

```diff
@@ -0,0 +1,86 @@
+import React from "react";
+import { Submission } from "@lib/responseDownloadFormats/types";
+
+export interface TableHeader {
+  title: string;
+  type: string;
+}
+
+export const AggregatedTable = ({
+  lang = "en",
+  headers,
+  submissions,
+}: {
+  lang?: string;
+  headers: TableHeader[];
+  submissions: Submission[];
+}) => {
+  const capitalizedLang = lang === "en" ? "En" : "Fr";
+
+  function formatAnswer(answer: string) {
+    return answer || "-";
+  }
+
+  return (
+    <table id={`responseTableRow${capitalizedLang}`} className="table-fixed text-left">
+      <thead>
+        <tr key="headers" className="flex">
+          {headers.map(({ title, type }, index) => (
+            <th
+              key={`heading-${title}-${index}`}
+              className={`${type === "dynamicRow" ? "w-120" : "w-64"} p-4 font-bold`}
+            >
+              {title}
+            </th>
+          ))}
+        </tr>
+      </thead>
+      <tbody>
+        {submissions.map((submission: Submission, index) => {
+          return (
+            <tr
+              key={submission.id}
+              className={`flex border-t-2 border-t-grey-300 ${
```

Comment:
`border-t-grey-300` doesn't exist. Probably due to the british spelling (grey vs gray) ... also be sure to check the custom tailwind config to make sure the shade you want is there